frontend/stylesheets/_variables.less

```
@@ -1,17 +1,24 @@
-@blue-base: #5660ff;
-@primary-color: #5660ff;
-@pastel-blue: #aabaff;
-@frosted-mint: #ddfdfa;
-@blue-zircon: #59f8e8;
-@black: rgba(0, 0, 0, 0.85);
 @left-menu-width: 360px;
-@xyRed: #eb4b98;
-@yzBlue: #5660ff;
-@xzGreen: #59f8e8;
-@footer-height: 20px;
-@background-blue-neurons: url(/assets/images/background_neurons_drawn.svg) 0 / cover no-repeat, @primary-color;
-@background-blue-organelles: url(/assets/images/background_mixed_cells_drawn.svg) 0 / cover no-repeat, @primary-color;
+@font-family: "Nunito", "Monospaced Number", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+  "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Helvetica Neue", Helvetica, Arial,
+  sans-serif;
+
+
 
 :root {
+    --color-wk-blue: #5660ff;
+    --color-black: rgba(0, 0, 0, 0.85);
+    --color-blue-zircon: #59f8e8;
+    --color-xy-viewport: #eb4b98;
+    --color-yz-viewport: var(--color-wk-blue);
+    --color-xz-viewport: var(--color-blue-zircon);
     --navbar-height: 48px;
+    --footer-height: 20px;
+}
+
+.ant-app {
+    // attach these variables to the antd <App> component otherwise these `--ant` variables are not in scope yet 
+    // the <App> component is expect to be there only one and very high level in the DOM tree
```

I find it hard to parse this comment. should it be
```suggestion
    // the <App> component is expected to be there only once and on a very high level in the DOM tree
```
?